code/modules/food_and_drinks/drinks/drinks/bottle.dm

```diff
@@ -160,6 +160,12 @@
 	icon_state = "badminka"
 	list_reagents = list(/datum/reagent/consumable/ethanol/vodka = 100)
 
+/obj/item/reagent_containers/food/drinks/bottle/vodka/special
+	name = "Strange vodka"
+	desc = "It's an odd mixture of an extremely strong alcohol, apparently originating in ancient Russia many, many years ago. It has a label -- '3,000% Alcohol, 2% Experimental Drugs!'"
+	icon_state = "badminka"
+	list_reagents = list(/datum/reagent/medicine/adminordrazine/strangevodka = 2, /datum/reagent/medicine/omnizine = 48, /datum/reagent/consumable/ethanol/vodka = 50)
```

Comment:
Split strangevodka into 20 0.1u pills for 20 instant full heals :grimacing:

Reply:
Will make strangevodka to 0.2 instead of 2u

Reply:
instead of lowering the amount he could make an make an early return if volume is too low.

Reply:
i have no idea how to do that, so the lowering of volume will work

Reply:
Taking a drink from the bottle at 2u would also have fulled healed you on second thought. Reducing it below that will probably cause issues with drink size, getting less than 0.1u when you take a swig (5u) and causing you to never get healed.

This is honestly a bit hard to balance properly, since the chem is inherently supposed to be unbalanced.